builder/src/main/java/cz/xtf/builder/db/MySQL.java

```diff
@@ -6,31 +6,31 @@
 public class MySQL extends AbstractSQLDatabase {
 
     public MySQL() {
-        super("MYSQL", "/var/lib/mysql/data");
+        super("MYSQL", "/var/lib/mysql");
```

Comment:
Could we make it configurable through the property? Something like: 
```
private static final String DATA_DIR_PROPERTY = "xtf.mysql.datadir";
private static final String MYSQL_DATA_DIR = System.getProperty(DATA_DIR_PROPERTY, "/var/lib/mysql/data");
...
public MySQL() {
        super("MYSQL", MYSQL_DATA_DIR);
}
```

Reply:
I haven't checked yet, but by looking at the code, it looks this would just change the mount point ... it would change the container run command accordingly e.g. `/usr/libexec/mysqld --initialize --datadir=/var/lib/mysql/data`

Reply:
@mnovak1 Tnks! that's a smart idea ... I tried and implemented it: please have a look

Reply:
It works with the oracle image (see eap-8.x-openshift-4-wildfly-openjdk17/502/)

Reply:
It also works with Red Hat image (see eap-8.x-openshift-4-wildfly-openjdk17/504/)

Reply:
In the end this PR would leave the current image setup unchanged, it would just add the option to customized the mount point for other MYSQL images